lib/DDG/Spice/SkyscannerFlightSearch.pm

```diff
@@ -0,0 +1,39 @@
+package DDG::Spice::SkyscannerFlightSearch;
+
+# ABSTRACT: This IA will call Skyscanner's API to retrieve the cheapest flight prices to 
+# the chosen destination
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+spice to => 'http://partners.api.skyscanner.net/apiservices/browseroutes/v1.0/GB/GBP/en-GB/$1/anywhere/anytime/anytime?apikey={{ENV{DDG_SPICE_SKYSCANNER_APIKEY}}}';
```

Comment:
Yep, I was looking into that for the next step with origin, destination and dates.
The issue here is that your service returns a slight different format of locale as far as I can see (en_GB) whereas our input is en-GB. I need to insure there is a 1-1 match for all locales (i.e. that we support all your possible values) and then replace the _ with -.
For the currencies I need to find a way to map the locale to the currency. We have an API endpoint that returns all the currencies that we support but I can't remember how we select currencies based on the locale. I'll ask my colleagues working on the APIs tomorrow morning. 

And, yes, lots of good examples in the repo - and I'm slowly getting my head around perl :-)